core/src/main/java/org/apache/brooklyn/core/catalog/internal/BasicBrooklynCatalog.java

```diff
@@ -417,6 +419,53 @@ else for (String key: otherKeys) {
         return result;
     }
 
+    public static Map<?,?> getCatalogMetadata(String yaml) {
+        Map<?,?> itemDef = Yamls.getAs(Yamls.parseAll(yaml), Map.class);
+        return getFirstAsMap(itemDef, "brooklyn.catalog").orNull();        
+    }
+    
+    public static VersionedName getVersionedName(Map<?,?> catalogMetadata) {
+        String id = getFirstAs(catalogMetadata, String.class, "id").orNull();
+        String version = getFirstAs(catalogMetadata, String.class, "version").orNull();
+        String symbolicName = getFirstAs(catalogMetadata, String.class, "symbolicName").orNull();
```

Comment:
@geomacy Yes it does clash with is the root cause of the issue I can see with my example blueprint. Why not simply use a field `bundle` for this?
```yaml
brooklyn.catalog
  bundle: org.apache.brooklyn.test
  version: 0.1.0
  id: test
  items:
  ...
```